packages/trendmicro/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@v8.4.0-rc1
```

```suggestion
    reference: git@v8.5.0
```
Let's try and bump it to 8.5 or above

---

what exact errors are you facing ? The integration will eventually have to pass for ecs 8.5 so better to get it fixed now.

---

With 8.5, I got this. 
```
Error: updating files failed: updating readme file README.md failed: rendering Readme failed: executing template failed: template: README.md:21:2: executing "README.md" at <fields "deep_security">: error calling fields: can't create fields validator instance (path: /mnt/c/k/current/trendmicro/integrations/packages/trendmicro/data_stream/deep_security): can't create field dependency manager: can't build fields schema: can't load fields: unmarshalling field body failed: yaml: unmarshal errors:Error: updating files failed: updating readme file README.md failed: rendering Readme failed: executing template failed: template: README.md:21:2: executing "README.md" at <fields "deep_security">: error calling fields: can't create fields validator instance (path: /mnt/c/k/current/trendmicro/integrations/packages/trendmicro/data_stream/deep_security): can't create field dependency manager: can't build fields schema: can't load fields: unmarshalling field body failed: yaml: unmarshal errors:
```